docs/debug_virt_stack/privileged-node-debugging.md

```diff
@@ -0,0 +1,225 @@
+# Privileged debugging on the node
+
+This article describes the scenarios in which you can create privileged pods and have root access to the cluster nodes.
+
+With privileged pods, you may access devices in `/dev`, utilize host namespaces and ptrace processes that are running on the node, and use the `hostPath` volume to mount nodes directly in the container.
+
+A quick way to verify if you are allowed to create privileged pods is to create a sample pod with the `--dry-run=server` option, like:
+
+```bash
+$ kubectl apply -f debug-pod.ymal --dry-run=server
+```
+
+# Build the container image
+
+KubeVirt uses [distroless containers](https://github.com/GoogleContainerTools/distroless) and those images don't have a package manager, for this reason it isn't possible to use the image as parent for installing additional packages.
+
+In certain debugging scenarios, the tools require to have exactly the same binary available. However, if the debug tools are operating in a different container, this can be especially difficult as the filesystem of the containers are isolated.
```

Comment:
```suggestion
In certain debugging scenarios, the tools require to have exactly the same binary available. However, if the debug tools are operating in a different container, this can be especially difficult as the filesystems of the containers are isolated.
```